### PR TITLE
update nu crates to 0.90.1

### PR DESCRIPTION
**Pull request by Inveracity**
This PR updates the nu crates to recently released 0.90.1

also, I've never written a single line of Rust until just now with this PR, so I have no clue if these changes are correct. But it compiles and the plugin seems to work, at least for sorting version numbers 🤞

### CODE REVIEW

File: src/plugin.rs [hunk 1]
```diff
@@ -95,7 +95,7 @@ impl SemverPlugin {
         match res {
             Ok(v) => Ok(v.into_value()),
             Err(e) => {
-                if ignore_errors {
```
Comment (abusch) [inline src/plugin.rs:98]: I think you instead need to do "throw" the error when getting the flag by doing `let ignore_errors = call.has_flag("ignore-errors")?;` above (note the `?` at the end), and then keep using the boolean here as before.

File: src/plugin.rs [hunk 2]
```diff
@@ -113,7 +113,7 @@ impl SemverPlugin {
             .map(VersionValue::try_from)
             .collect::<Result<Vec<_>, _>>()?;
         versions.sort_by(|a, b| a.deref().cmp(b.deref()));
-        if reverse {
+        if reverse.is_ok() {
```
Comment (abusch) [inline src/plugin.rs:116]: Same here.

**Review by abusch — CHANGES_REQUESTED** [review]
Thanks for the PR! I've left a couple of comments about flag handling, but otherwise it looks good!

File: src/plugin.rs (hunk 1 above)
Comment (Inveracity) [reply src/plugin.rs]: ah, that's a cool language feature. I've updated the code, I hope it's right. Thank you!

**Pull request merged**